go.mod

```diff
@@ -33,24 +33,24 @@ require (
 	golang.org/x/tools v0.0.0-20200916195026-c9a70fc28ce3 // indirect
 	gopkg.in/src-d/go-git.v4 v4.13.1
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.18.8
-	k8s.io/apiextensions-apiserver v0.18.6
-	k8s.io/apimachinery v0.18.8
+	k8s.io/api v0.20.4
+	k8s.io/apiextensions-apiserver v0.20.1
+	k8s.io/apimachinery v0.20.4
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20200410145947-bcb3869e6f29
+	k8s.io/kube-openapi v0.0.0-20201113171705-d219536bb9fd
 	knative.dev/eventing v0.18.0
 	knative.dev/pkg v0.0.0-20200922164940-4bf40ad82aab
-	sigs.k8s.io/controller-runtime v0.6.3
+	sigs.k8s.io/controller-runtime v0.6.1
```

Comment:
Why downgrade here?

Reply:
undo it. To setup complatablity I had made changes in many pkg version. I might miss to reset it